firefox-ios/firefox-ios-tests/Tests/XCUITests/TodayWidgetTests.swift

```diff
@@ -11,10 +11,11 @@ let editHomeScreenButton = "com.apple.springboardhome.application-shortcut-item.
 let removeWidgetButton = "com.apple.springboardhome.application-shortcut-item.remove-widget"
 
 // Widget Buttons Identifier
-var goToCopiedLink = springboard.buttons["Go to Copied Link"]
-var newPrivateSearch = springboard.buttons["New Private Search"]
-var newSearch = springboard.buttons["New Search"]
-var clearPrivateTabs = springboard.buttons["Clear Private Tabs"]
+// TODO FXIOS-12604 These global properties are not concurrency safe
```

Comment:
@mdotb-moz @dragosb01 Heads up: The devs are working on Swift migration. More updates are coming.